### ISSUE TITLE
[Linear] Missing Projects in "Search Projects"

### ISSUE THREAD

**Issue by neobarthold** [raycast/extensions#15052]
### Extension

https://www.raycast.com/linear/linear

### Raycast Version

1.84.3

### macOS Version

15.0.1

### Description

Some projects are missing in the list if I use the "Search Projects" command, and I can't see any reason why. These projects are active and have a status that other projects, which are listed, also have.

### Steps To Reproduce

Open the command, search for a project.

### Current Behaviour

Some projects are missing in the list.

### Expected Behaviour

Every project should be in the list.

**Comment by thomaslombart** [comment]
Indeed. There's no support for pagination in the `Search Projects` command. Would you like to work on it? If yes, here's our [guide on extension contributions](https://developers.raycast.com/basics/contribute-to-an-extension) so that you can open up a pull request.
